boards/adi/max32662evkit/Kconfig.defconfig

```diff
@@ -20,7 +20,7 @@ choice LV_COLOR_DEPTH
 endchoice
 
 config LV_COLOR_16_SWAP
-	default y
+	default y if LV_COLOR_DEPTH_16
```

Comment:
what about using [`configdefault`](https://docs.zephyrproject.org/latest/build/kconfig/extensions.html#kconfig-extensions) ?

Reply:
Very good call, forgot that this exists

Reply:
Wasn't aware of this- I'll update the PR to use this, thanks!